OrbitLinuxTracing/include/OrbitLinuxTracing/TracingOptions.h

```diff
@@ -0,0 +1,21 @@
+#ifndef ORBITLINUXTRACING_INCLUDE_ORBITLINUXTRACING_TRACINGOPTIONS_H_
+#define ORBITLINUXTRACING_INCLUDE_ORBITLINUXTRACING_TRACINGOPTIONS_H_
+
+namespace LinuxTracing {
+enum UnwindingMethod {
+  kUndefined = 0,
```

Comment:
While having the `kUndefined` value as the first value might be common, it doesn't really mean anything for us and is something else to check for. I would replace the pair `trace_callstacks`,`unwinding_method` with a single enum called `SamplingMethod` (from the point of view of `perf_event_open`, this is how we sample, not how we unding), have `kOff` instead of `kUndefined`, and give it the meaning of `trace_callstacks == false`.

Reply:
Sounds good to me!

Reply:
I like that idea and will change it accordingly.